tests/test_spectrometry.py

```diff
@@ -91,8 +93,33 @@ def test_gross_counts():
 def test_net_count():
     nc=sa.net_counts(gspec1, 475, 484, 1)
 
-
-
+def test_net_area():
+    nc=sa.net_area(gspec1, 475, 484)
+    #assert_equal(nc, 20454.5)
+
+def test_end_point_average_area():
+    nc = sa.end_point_average_area(gspec1,475,484,var=5)
+    #assert_equal(nc,20355.5) 
```

Comment:
Shouldn't the tests here have an assert? Is the problem that you are doing a float comparison? You should probably use `assert_almost_equals` in this case